dataprofiler/labelers/data_processing.py

```diff
@@ -2079,3 +2079,113 @@ def process(self, data, labels=None, label_mapping=None, batch_size=None):
         # predictions is the argmax of the average of the cell's label votes
         results["pred"] = np.argmax(results["pred"], axis=1)
         return results
+
+
+class ColumnNameModelPostprocessor(
+    BaseDataPostprocessor, metaclass=AutoSubRegistrationMeta
+):
+    """Subclass of BaseDataPostprocessor for postprocessing regex data."""
+
+    def __init__(self, parameters):
+        """Initialize the ColumnNameModelPostProcessor class."""
+        # set parameters
+        if not parameters:
+            parameters = {}
+        parameters.setdefault("true_positive_dict", None)
```

Comment:
do we want these defaults to both be `None`, will that error by default or do we want to use any defaults?

Reply:
- `parameters.setdefault("true_positive_dict", None)` default for this is a no go in my opinion unless we want "out of the box" examples of attributes and labels, which I'd say is out of scope for this PR
- `parameters.setdefault("positive_threshold_config", None)`: however a default for this is a fair point, but really only matters if the `true_positive_dict` is set as well...